llarp/net/route.cpp

```diff
@@ -449,14 +461,18 @@ namespace llarp::net
       }
     });
     return gateways;
-#elif __APPLE__
+#elif __APPLE__ || __sun
     LogDebug("get gateways not on ", ifname);
     const auto maybe = GetIFAddr(ifname);
     if (not maybe.has_value())
       return gateways;
     const auto interface = maybe->toString();
     // mac os is so godawful man
+#if __APPLE__
     FILE* p = popen("/usr/sbin/netstat -rn -f inet", "r");
+#else
+    FILE* p = popen("/usr/bin/netstat -rn -f inet", "r");
```

Comment:
being a mashup of BSD and SysV works well here: macOS and Solaris have the same flavour of `netstat(1M)`